sources/packages/api/src/route-controllers/education-program/models/save-education-program.dto.ts

```diff
@@ -8,9 +8,44 @@ import {
   getISODateOnlyString,
   getUserFullName,
 } from "../../../utilities";
+
 /**
- * Dto that represents education programs form object.
+ * Dto that represents education program with offering form object.
  */
+export class EducationProgramWithOfferingsAPIOutDTO {
```

Comment:
The DTO is 97% percent about the program itself, I believe that we can name it just as it was before `EducationProgramAPIOutDTO`.